README.md

```diff
@@ -1,29 +1,36 @@
-# React Demo App
+# Cognite Glimpse
 
-This repository shows how to bootstrap a [React]-based application at Cognite.
-It demonstrates best practices, such as:
+This repository is a showcase of how to build apps on top of [Cognite Data Fusion](https://cognite.com/cognite/cognite-data-fusion/developers/). This app is mainly useful for manufacturing operators, To perform the CNC machines as efficiently as possible and to understand when they are not performing at their best.
 
-- Lint setup to use
-- Jenkinsfile steps
-- TypeScript usage
-- Folder / component layout
-- ...
+This app is basically a widget container that shows the CNC machine's details. Widgets can be added, deleted (customized), and moved by dragging and dropping into an empty position.
```

Comment:
Information is presented using a set of widgets which the operator can configure. Each widget is laid out in a fixed sized grid. Position of widgets in the grid can be changed based on operator's preferences. Overall dashboard configuration is persisted for each user separately.